<launch>
	<arg name="bag" default="test_data.bag"/>
	
	<node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find ford_msgs)/config/rviz.rviz"/>
	<node name="ped_manager" pkg="ford_msgs" type="ped_manager_node" output="screen">
		<remap from="~clusters" to="/cluster/output/clusters" />
		<remap from="~ped_id" to="/pedestrian_cluster_id" />
		<!-- Parameters -->
		<param name="~diff_pub_period" value="1.0"/>
		<param name="~prune_period" value="10.0"/>
		<param name="~inactive_tol" value="5.0"/>
		<param name="~vis_period" value="0.03"/>
		<param name="~dump_period" value="10.0"/>
		<param name="~recent_period" value="0.2"/>
		<param name="~recent_tol" value="10.0"/>
	</node>
	<!-- <node name="ped_logger" pkg="ford_msgs" type="ped_logger.py" output="screen"> -->
		<!-- <remap from="~ped_diff" to="ped_manager/ped_diff"/> -->
	<!-- </node> -->
	<node name="ped_pred" pkg="acl_gp" type="predictor_node" output="screen">
		<remap from="~input" to="/ped_manager/ped_recent"/>
	</node>

	<param name="use_sim_time" value="true"/>
	<node pkg="rosbag" type="play" name="bag_player" args="--clock $(find ford_msgs)/bag/$(arg bag) --rate=1.0"/>
</launch>